drivers/src/fuse_log.rs

```diff
@@ -21,12 +21,12 @@ pub enum FuseLogEntryId {
     Invalid = 0,
     VendorEccPubKeyIndex = 1,       // 4 bytes  (From Manifest)
     VendorEccPubKeyRevocation = 2,  // 4 bytes  (From Fuse)
-    ManifestFmcSvn = 3,             // 4 bytes
+    DeprecatedManifestFmcSvn = 3,   // 4 bytes
```

Comment:
nit: Should the deprecated values have a `_` prefix to denote it isn't used? Also you can mark fields as `#[deprecated]` to get the compiler to emit a warning if they're used.

If you still want the `From` impl to be exhaustive, you can add `#[allow(deprecated)]`. Or you could just make the deprecated entries return `Invalid`

Reply:
Done.